data/cosmo.py

```diff
@@ -7,6 +7,8 @@
 import numpy as np
 import tensorflow as tf
 
+import horovod.keras as hvd
```

Comment:
Hmm, does this need to be `import horovod.tensorflow.keras as hvd`?
there can be differences in the keras versions in/out of TF.
Actually in this case it may not matter because you're just using the `local_rank` and `local_size` functions, which should come from horovod core functionality.

I can test it.